### ISSUE TITLE
Fix Time Period filter - 508 Issue 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#1492]
## Required Fix 

- [ ] RF-11
- [x] RF-12
- [x] RF-13

## Context 
See [EPA CAMD Accessibility Report ](https://teams.microsoft.com/l/file/31E8B6EF-1313-4495-B867-46347BB7E4A2?tenantId=88b378b3-6748-4867-acf9-76aacbeca6a7&fileType=docx&objectUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam%2FShared%20Documents%2FDatateers%2F5-UI%20Design%2F02%20508%20and%20Accessibility%2FEPA.CAMPD.Accessibility.Report%20.docx&baseUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam&serviceName=teams&threadId=19:f4349e2ba95e43f3b29e8208bc8f5d5f@thread.skype&groupId=21a5e1cd-d3f1-48a6-801f-0d34e2d63d23) for how to fix and additional details 

| Issue #  | Issue Description  |
|-|-|
| RF-11 | The messages shown to the user as a result of input errors in form fields on the page are not accessible to Assistive Technology. This issue is a violation of Section 508 and of Success 3.3.1 in WCAG 2.0. |
| RF-12 | There is a label associated with the form field but the label itself is not descriptive or accurate. This issue is a violation of WCAG 2.0 Success Criterion 2.4.6. |
| RF-13 | Generic link text such as ‘Click Here’, ‘Read More’, etc. is used, and there are no clues to link destination or function in the text immediately surrounding the link. This issue is a violation of WCAG 2.0 Success Criterion 2.4.4 |

## Testing 
Tester: @aprematta

**Comment by aprematta** [comment]
Testing complete, RF 12 and 13 are good. RF-11 is still an issue. I can see it's in a live region however neither screenreader I use is announcing the error message when it displays. The check marks are also still inaccessible. I see that aria checked is used but this os only valid for form selections.

**Comment by JanellC** [comment]
Issue RF-11 covered in Bug: Make Time Period Error Message Screen Reader Compatible #1549
